packages/orm/src/EntityManager.ts

```diff
@@ -139,24 +147,15 @@ export type LoaderCache = Record<string, DataLoader<any, any>>;
 type FilterAndOrder<T> = [FilterOf<T>, OrderOf<T> | undefined];
 
 export class EntityManager {
-  constructor(public knex: Knex) {}
+  constructor(public knex: Knex) {
+    this.contexty.create();
```

Comment:
Hm, my understanding is that we'd call this `.create()` just once as a module-level constant, b/c it installs a "static" hook into the async_hooks runtime...

Reply:
Separate from moving the `new Contexty()` up to be a module-level const, do we need this `.create()` call in the constructor? Maybe but just want to understand.

Reply:
We don't need it necessarily, but it makes it so that code checking the context doesn't need to worry about there not being one yet.